english/alchemy/alchemy.md

```diff
@@ -38,5 +38,7 @@ You slip the ring over your finger and ...
 
 [Transform!](../transform/transform.md)
 
+[You teleport to the Incantation Forrest](../incantation/incantation-forrest.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.